src/hotel/src/service/hotelService.js

```diff
@@ -0,0 +1,43 @@
+var hotels = require(process.env.INIT_CWD +"/hotel-data.json");
+var hotelInfo = require(process.env.INIT_CWD +"/hotel-info.json");
+
+function getHotels(city, country, f) {
```

Comment:
There are duplicate hotels for cities but with different prices. Is that intentional?

Reply:
yes. think of hotels in primo locations.